src/screens/LoadWalletErrorScreen.js

```diff
@@ -10,7 +10,7 @@ import {
   Text,
   View,
 } from 'react-native';
-import { useDispatch, useSelector } from 'react-redux';
+import { useDispatch } from 'react-redux';
```

Comment:
`useSelector` is not being used on this file so I removed it to avoid a warning on the linter, not related to this PR